operating/manage-compliance/managing-compliance-10.adoc

```diff
@@ -68,6 +68,12 @@ include::modules/view-compliance-status-for-namespaces.adoc[leveloffset=+2]
 
 include::modules/view-compliance-status-for-standard.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+[id="additional-resources_manage-security-policies"]
+==== Additional resources
```

Comment:
I think you want to use a trailing include here. Typically, the additional resources for a whole assembly comes at the end of the assembly.